helm/alfresco-content-services/values.yaml

```diff
@@ -391,29 +391,123 @@ alfresco-infrastructure:
   nginx-ingress:
     enabled: false
 
-alfresco-search:
+alfrescosearch:
   enabled: true
-# If enabled is set to false, then external host and port need to point to the external instance of SOLR6, and in this case:
-# Note: Rule_05-network-policy-search will be disable.
-# Note: Rule_04-network-policy-repository internal trafic to SOLR6 instance will be disabled.
-# Note: ingress-repository will not block external trafic to */solr/*
-#  external:
-    # Host dns/ip of the external solr6 instance.
-#    host: "127.0.0.1"
-    # Port of the external solr6 instance.
-#    port: "8983"
+  # Default values for alfrescosearch.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+
+  # Define the alfrescosearch properties to use in the k8s cluster
+  # This is the search provider used by alfresco-content-repository
+  replicaCount: 1
+  # Define the type of Alfresco Search to use. The default is Alfresco Search Services.
+  # The type can be set to use Insight Engine with --set alfrescosearch.type="insight-engine",alfrescosearch.global.alfrescoRegistryPullSecrets="quay-registry-secret",alfresco-insight-zeppelin.enabled="true"
+  # As the Docker Image for Insight Engine is not publicly available the alfrescoRegistryPullSecrets has to be set
+  # More information can be found on https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md
+  type: "search-services"
   searchServicesImage:
-    tag: "1.4.2"
+    repository: alfresco/alfresco-search-services
+    tag: "1.4.1"
+    pullPolicy: &searchDockerImagePullPolicy Always
+    internalPort: &searchDockerImageInternalPort 8983
+  insightEngineImage:
+    repository: quay.io/alfresco/insight-engine
+    tag: "1.4.1"
+    pullPolicy: *searchDockerImagePullPolicy
+    internalPort: *searchDockerImageInternalPort
+  service:
+    name: solr
+    type: ClusterIP
+    externalPort: 80
+  ingress:
+    enabled: false
```

Comment:
It's probably also worth including the original comment about ingress being disabled by default here and including the link to the documentation.

Reply:
Added original comments